GitUI/CommandsDialogs/FormCommit.cs

```diff
@@ -391,6 +391,8 @@ void AddToSelectionFilter()
                         }
                     });
 
+            UICommands.PostRepositoryChanged += UICommands_PostRepositoryChanged;
```

Comment:
Could you please confirm there's no double assignment of the handler? That is, that line:616 won't get executed before or after this line.

Reply:
This is the same pattern already used in `FormBrowse`, so if one is wrong, they're both wrong.

However, it might be wrong, given that the method is called from the `GitModuleForm.UICommands` setter, which is called from the `GitModuleForm` constructor, which would theoretically be called from the child class's constructor. But I need someone else to look at `FormBrowse`'s implementation to be certain.

Reply:
It *is* handled by `FormCommit.OnUICommandsChanged` (see below).